tools/accuracy_checker/accuracy_checker/evaluators/custom_evaluators/README.md

```diff
@@ -26,15 +26,12 @@ Optionally you can provide `module_config` section which contains config for cus
 * **Sequential Action Recognition Evaluator** demonstrates how to run Action Recognition models with encoder + decoder architecture.
   [Evaluator code](sequential_action_recognition_evaluator.py)
   Configuration file examples:
-    * [action-recognition-0001-encoder](../../../configs/action-recognition-0001-encoder.yml) - running full pipeline of action recognition model.
-    * [action-recognition-0001-decoder](../../../configs/action-recognition-0001-decoder.yml) - running only decoder stage with dumped embeddings of encoder.
+    * [action-recognition-0001](../../../configs/action-recognition-0001.yml) - running full pipeline of action recognition model.
```

Comment:
There's no reason to have a list anymore if there's only one entry. You can just say "Configuration file example: \[action-recognition-0001](../../../configs/action-recognition-0001.yml)".

Reply:
Done